src/js/components/awardv2/idv/amounts/charts/ExceedsPotentialChart.jsx

```diff
@@ -7,8 +7,6 @@ import React from 'react';
 
 import { generatePercentage } from 'helpers/aggregatedAmountsHelper';
 
-import InfoTooltip from 'components/awardv2/idv/InfoTooltip';
-import { awardAmountsExtremeOverspendingInfo } from 'components/awardv2/idv/InfoTooltipContent';
```

Comment:
Do we want to remove `awardAmountsExtremeOverspendingInfo` from  `components/awardv2/idv/InfoTooltipContent`?

Reply:
@jonhill13 yes.